### PR TITLE
feat(docs): consolidate `single` commands

### PR DESCRIPTION
**Pull request by kanadgupta**
| 🚥 Fix RM-5580 |
| :-- |

## 🧰 Changes

This removes our `:single` commands and extends the functionality of the `changelogs`, `custompages`, and `docs` commands to intelligently determine whether the input is a directory vs. a single file.

The following commands have been removed in favor of their standard counterparts:

- [x] `changelogs:single`
- [x] `custompages:single`
- [x] `docs:single`

## 🧬 QA & Testing

I've left the test suites separated out so single file tests vs. directory tests are in separate files. Do they pass?

### CODE REVIEW

**Comment by kanadgupta** [comment]
@garrett-wade FYI!

File: __tests__/cmds/changelogs/index.test.ts [hunk 1]
```diff
@@ -37,21 +37,23 @@ describe('rdme changelogs', () => {
     delete process.env.TEST_CI;
   });
 
-  it('should error if no folder provided', () => {
-    return expect(changelogs.run({ key })).rejects.toThrow(
-      'No folder provided. Usage `rdme changelogs <folder> [options]`.'
+  it('should error if no path provided', () => {
+    return expect(changelogs.run({ key })).rejects.toStrictEqual(
```
Comment (erunion) [inline __tests__/cmds/changelogs/index.test.ts:41]: Why not do `.toThrow` here anymore?

Comment (kanadgupta) [reply __tests__/cmds/changelogs/index.test.ts:41]: `.toThrow` is technically a partial match on `err.message`, `.toStrictEqual(<full error object>)` is a stricter check

File: src/cmds/docs/index.ts [hunk 2]
```diff
@@ -1,35 +1,32 @@
...
     this.position = 1;
 
-    this.hiddenArgs = ['folder'];
+    this.hiddenArgs = ['filePath'];
     this.args = [
       this.getKeyArg(),
       this.getVersionArg(),
       {
-        name: 'folder',
+        name: 'filePath',
```
[the remark this answers is not in the document]
Comment (kanadgupta) [reply src/cmds/docs/index.ts:29]: Both! This variable isn't user-facing since it's the default arg, but I wasn't sure what to call it.

**Review by erunion — APPROVED** [review]

**Pull request merged**